brambling/admin/forms.py

```diff
@@ -42,7 +42,7 @@ class PersonChangeForm(forms.ModelForm):
 
     class Meta:
         model = Person
-        fields = ('email', 'password', 'is_active',)
+        fields = ('first_name','last_name','email', 'password', 'is_active',)
```

Comment:
These shouldn't need to be added here, since they weren't mentioned before either. Are you sure they should be here? If not, we should probably not add them. (Also, code style note: always put a space after a comma.)